examples/web_api.rs

```diff
@@ -1,89 +1,99 @@
-#![deny(warnings)]
 extern crate futures;
 extern crate hyper;
 extern crate pretty_env_logger;
 extern crate serde_json;
 
 use futures::{future, Future, Stream};
+use serde_json::{from_str, Value, to_string};
```

Comment:
I feel these names are a bit too generic to be used without qualifying them (`to_string("foo")` sounds like `"foo".to_string()`), what if they were used qualified instead (so, like `serde_json::to_string("foo")`)?

Reply:
Fair enough.